src/compilation_context.ml

```diff
@@ -125,7 +125,7 @@ let for_alias_module t =
         ["-w"; "-49"; "-nopervasives"; "-nostdlib"]
   ; includes     = Includes.empty
   ; alias_module = None
-  ; stdlib       = None
+  ; stdlib       = None (* FIXME does this do anything? it appears not*)
```

Comment:
It's used in module_compilation.ml

Reply:
Hmm, I tried removing it and it didn't affect the tests. I'll remove this comment for now.

Reply:
Yh, it's not tested. It's part of `(using experimental_building_ocaml_compiler_with_dune 0.1)`